wolfhsm/wh_comm.h

```diff
@@ -170,8 +177,8 @@ typedef struct {
     void* transport_context;
     const whTransportServerCb* transport_cb;
     uint16_t reqid;
-    uint8_t packet[WOLFHSM_COMM_MTU];
-    whHeader* hdr;
+    uint8_t packet[WH_COMM_MTU];
+    whCommHeader* hdr;
     uint8_t* data;
     uint32_t client_id;
     uint32_t server_id;
```

Comment:
same as above, can we make the "context" explicit in the struct name?

Reply:
fixed